frontend/javascripts/oxalis/store.ts

```diff
@@ -570,7 +570,7 @@ export type OxalisState = {
   readonly localSegmentationData: Record<
     string, //layerName
     {
-      readonly meshes: Record<string, Record<number, MeshInformation>>; //string represents additional coordinates, number is the segment ID
+      readonly meshes: Record<string, Record<number, MeshInformation> | undefined> | undefined; //string represents additional coordinates, number is the segment ID
```

Comment:
please add a comment that explains that `undefined` was added to enforce stricter null checks when using these structures.